tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
```

Comment:
Opened, reopened, and synchronize are the default pull_request trigger activity types, so unless we are adding additional activity types (like labeled/unlabeled, which we currently use for the changelog workflow and that is something we might want to consider) these are not needed.